cvmfs/catalog_diff_tool.h

```diff
@@ -52,6 +52,9 @@ class CatalogDiffTool {
   bool Run(const PathString& path);
 
  protected:
+  virtual bool IsIgnoredPath(const PathString& path) { return false; }
```

Comment:
Let's add comments that explain the semantics of an ignored part and a reportable part.  On first sight, they seem to be complementary but only in the merge tool it becomes apparent that there are subtle differences.

Reply:
Good idea.  I've added explanatory comments to the prototypes, along with a concrete example of the situation in which a path is not ignored but also not reportable.